src/components/App.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import Display from './Display';
 import ButtonPanel from './ButtonPanel';
+import calculate from '../logic/calculate'; // eslint-disable-line no-unused-vars
```

Comment:
- [ ] Remove the `eslint-disable-line no-unused-vars`. And place <calculate /> inside the react.fragment short syntax.